src/com/magento/idea/magento2plugin/actions/generation/dialog/NewEntityDialog.form

```diff
@@ -137,12 +143,19 @@
               </vspacer>
             </children>
           </grid>
-          <grid id="e3309" layout-manager="GridLayoutManager" row-count="8" column-count="2" same-size-horizontally="false" same-size-vertically="false" hgap="-1" vgap="-1">
+          <grid id="e3309" binding="uiComponentsPanel" layout-manager="GridLayoutManager" row-count="8" column-count="2" same-size-horizontally="false" same-size-vertically="false" hgap="-1" vgap="-1">
             <margin top="10" left="10" bottom="0" right="10"/>
             <constraints>
               <tabbedpane title="Admin UI Components"/>
             </constraints>
-            <properties/>
+            <properties>
+              <background color="-12828863"/>
```

Comment:
I think this is the reason of having that style issue

Reply:
Thank you! I'll take a look.